### PR TITLE
fix: Listing component spacing

### PR DESCRIPTION
**Pull request by g10**
## Description

Fix spacing / padding in Listing component

**Fixes** # 17308

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 23.04.x (master)

<h2> How this pull request can be tested ? </h2>

Storybook → Listing → With Short Label Columns

### CODE REVIEW

File: centreon/packages/ui/src/Listing/index.tsx [hunk 1]
```diff
@@ -515,6 +515,9 @@ const Listing = <TRow extends { id: RowId }>({
 
   const areColumnsEditable = not(isNil(onSelectColumns));
 
+
+
+
```
Comment (Thebarda) [inline centreon/packages/ui/src/Listing/index.tsx:520]:
```suggestion
```

**Review by Thebarda — COMMENTED** [review]
https://user-images.githubusercontent.com/12515407/233356970-f16dece2-002a-4892-a6d3-d37bfb09be3f.mov

When dragging a column, the background is missing.
I think, you need to add the following block of code in the Listing/index.tsx file
```
'.listingHeader > div > div': {
  backgroundColor: theme.palette.background.listingHeader,
  alignItems: 'center',
  display: 'flex',
},
```

**Review by Thebarda — APPROVED** [review]

**Review by bdauria — DISMISSED** [review]

**Comment by g10** [comment]
fyi. additional commit fixes the size issue of the draggable header

File: centreon/packages/ui/src/Listing/Header/index.tsx [hunk 2]
```diff
@@ -196,6 +196,7 @@ const ListingHeader = ({
           Content={Content}
           additionalProps={[sortField, sortOrder]}
           collisionDetection={closestCenter}
+          hasKeyboardInputEnabled={false}
```
Comment (Thebarda) [inline centreon/packages/ui/src/Listing/Header/index.tsx:199]: Why you disabling keyboard inputs?

**Comment by bdauria** [comment]
@g10 it might be a good idea to check with @Wedierre if he could provide you with an image for the empty result case to add some emotional design (for a later stage though)

File: centreon/packages/ui/src/Listing/index.tsx [hunk 3]
```diff
@@ -661,26 +665,46 @@ const Listing = <TRow extends { id: RowId }>({
...
+                        isRowHovered={false}
+                      >
+                        <LoadingSkeleton
+                          height={dataStyle.body.height}
+                          variant="text"
+                          width="100%"
+                        />
+                      </Cell>
+                    </TableRow>
+                  ))
```
[the remark this answers is not in the document]
Comment (g10) [reply centreon/packages/ui/src/Listing/index.tsx]: has been extracted / DRYd up the code

**Review by Thebarda — DISMISSED** [review]

**Review by bdauria — DISMISSED** [review]

**Review by Thebarda — APPROVED** [review]

**Comment by Thebarda** [comment]
@g10 There are some breaking changes in this PR

**Comment by g10** [comment]
@Thebarda seemed an issue with rebase on the `develop` branch from yesterday, now it should be clean / no breaking changes

**Review by bdauria — APPROVED** [review]

**Comment by bdauria** [comment]
@g10 you can now merge this PR

**Pull request merged**